steam_buddy/server.py

```diff
@@ -444,6 +444,20 @@ def mangohud():
 	finally:
 		redirect('/')
 
+@route('/type')
+@authenticate
+def type():
+    return template('type.tpl')
+
+@route('/type/string', method='POST')
+@authenticate
+def type_string():
+    str = sanitize(request.forms.get('str'))
+    try:
+        subprocess.call(["xdotool", "type", "--", str])
```

Comment:
Does this actually work? Last time I tried xdotool it had stopped working. For example, toggling mangohud isn't working and I don't know why.

Reply:
At least calling it over ssh works, in this case in the steam browser:
```
[gamer@gameros ~]$ export DISPLAY=:0.0
[gamer@gameros ~]$ xdotool type -- foo
```